fdbclient/NativeAPI.h

```diff
@@ -129,6 +129,7 @@ class Cluster : public ReferenceCounted<Cluster>, NonCopyable {
 private: friend class ThreadSafeCluster;
 		 friend class AtomicOpsApiCorrectnessWorkload; // This is just for testing purposes. It needs to change apiVersion
 		 friend class AtomicOpsWorkload; // This is just for testing purposes. It needs to change apiVersion
+		 friend class VersionStampWorkload; // This is just for testing purposes. It needs to change apiVersion
```

Comment:
I think this is reasonable given the precedent, but when we do come up with a better API version testing story I hope we aren't relying on having a big list of friend classes here.